### ISSUE TITLE
Create additional user groups

### ISSUE THREAD

**Issue by ghost** [OpenWork-NZ/core-geonetwork#82]
Can we please change my permissions, so I have the ability to create and edit user groups. Currently we have have AntarcticNZ and Sample Group. We don't require Sample Group, and there are additional groups I would like to have the permissions to create.

**Comment by ByronCinNZ** [comment]
Creation of groups requires admin permissions. All other users are created in reference to groups. Perhaps we could provide a log in that would be used for this purpose. But this would expose all admin functionality and would be best used very sparingly and for specific purposes.

**Comment by ghost** [comment]
This hasn't been implemented.

Before the change there were two groups under “Manage groups” - “AntarcticaNZ” and “Sample Group”. Sample Group has now been removed (as requested) but the new groups haven’t been added (Antarctic Science Platform, National Modelling Hub, Antarctic Heritage Trust).
 
 
![image](https://user-images.githubusercontent.com/91862430/232292975-85453817-4fcd-47b8-b564-ec3cefd5eeab.png)

 
And we an option in the drop down when creating a new group.
 
![image](https://user-images.githubusercontent.com/91862430/232293042-98c6e47e-0b93-4072-8e1b-e8a1e0f2b531.png)

 
These groups then appeared under “PROVIDED BY”.
 
![image](https://user-images.githubusercontent.com/91862430/232293102-c5a86955-859c-4d15-bea5-c7e1bcf9818b.png)
 
Instead, these groups appear under “VISIBLE BY”.

Could we please add these with consistent naming. Are hyphens required? And spaces between words would be consistent so “AntarcticaNZ” would be “Antarctica NZ”.

**Comment by ByronCinNZ** [comment]
This problem was that your login only allowed access to the one group. 
Now you have access to all groups.

**Comment by ghost** [comment]
Is it possible for these to appear in the facet pane under "PROVIDED BY" where it would be available to users who aren't signed in? The way it is working now is that there is no way to filter down on these records by Provider.

![image](https://user-images.githubusercontent.com/91862430/232365275-4cf32715-c343-4195-9d77-1dc7c8b81cce.png)

**Comment by ByronCinNZ** [comment]
I would have to double check, but I believe that the “Provied by” facet is for data harvested from different servers.

What is the behaviour you would like it to have? Provided by different groups?

Cheers,
Byron

**Comment by ghost** [comment]
Yes you're correct. They're the different data centres who provide to the Italian Antarctic Data Center.

This is low priority and would be tasked after the high priority items (appearing in order): 
https://github.com/OpenWork-NZ/core-geonetwork/issues/95
https://github.com/OpenWork-NZ/core-geonetwork/issues/97
https://github.com/OpenWork-NZ/core-geonetwork/issues/99
https://github.com/OpenWork-NZ/core-geonetwork/issues/98

Yes please. However, I would like to use a different name than "Group". If possible my preference would be "Organisation Provider" which would represent each of the Groups (Antarctica New Zealand, Antarctic Science Platform, National Modelling Hub, Antarctic Heritage Trust). These groups would appear in the facet pane which is available to anyone who visits the catalogue as a quick filter to isolate records.
